src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,46 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
+        System.out.print("Console calculator:\n " +
+                "Enter 'add' to perform addition.\n " +
+                "Enter 'sub' to perform subtraction.\n " +
+                "Enter 'mul' to perform multiplication.\n " +
+                "Enter 'div' to perform division.\n " +
+                "Enter 'exit' to exit.\n " +
+                "Enter 'help' to see help message.\n ");
+        int exit=1;
+
+        while (exit==1){
+            System.out.print("Make your choice.\n");
+            Scanner sc = new Scanner(System.in);
+            String s1 = sc.nextLine();
+            if (s1.equals("exit")) {System.out.println("Bye-bye.");
```

Comment:
There is actually no need for this check. It'll be better if you could move all possible commands cases into `switch` statement. And also do not forget about code duplication - create additional functions if they'll help you.